README.md

```diff
@@ -282,7 +282,7 @@ And finally, you are ready to go! Or at least you should be...
 **Iris Mestres** | [Contact me][deliaBlue]
 
 ### Junior Contributors
-
+**Paula Martín** | [Contact me][paulamartin14] | I haven't slept much tonight
 
 
 
```

Comment:
Istead of [Contact me][paulamartin14] you should have [paulamartin14][paulamartin14] so we can see you user name.